Makefile

```diff
@@ -27,6 +27,9 @@ clean: ## Remove all the containers along with volumes
 build: ## Build the Docker image (ignoring cache)
 	docker build -f dev/Dockerfile . -t astronomer-providers-dev:latest --no-cache
 
+build-emr_eks_container_example_dag-image: ## Build the Docker image for EMR EKS containers example DAG (ignoring cache)
```

Comment:
Instead of only emr_eks what about doing for integration test i.e all examples?

Reply:
good idea, we can do it in a separate PR as this issue & PR is related to EMR-EKS DAG failure.